src/models.ts

```diff
@@ -79,16 +80,30 @@ export const SortFieldDisplayName: {
   [key in SortField]: string;
 } = {
   relevance: 'Relevance',
-  alltimeview: 'All-time Views',
-  weeklyview: 'Weekly Views',
+  alltimeview: 'All-time views',
+  weeklyview: 'Weekly views',
   title: 'Title',
-  datearchived: 'Date Archived',
-  date: 'Date Published',
-  datereviewed: 'Date Reviewed',
-  dateadded: 'Date Added',
+  date: 'Date published',
+  datearchived: 'Date archived',
+  datereviewed: 'Date reviewed',
+  dateadded: 'Date added',
   creator: 'Creator',
```

Comment:
(Switching from Title Case to Sentence case)

Reply:
Funny. Never heard it called "Sentence Case" (but makes total sense!).